build.cmd

```diff
@@ -3,5 +3,8 @@
 REM Copyright (c) .NET Foundation and contributors. All rights reserved.
 REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+call %~dp0init-tools.cmd %*
+if %errorlevel% neq 0 exit /b %errorlevel%
```

Comment:
With my change, you won't need this anymore. I added a init-tools.ps1 that build.ps1 is invoking already.